install

```diff
@@ -58,7 +58,6 @@ do
             ;;
         *)  # unknown option
             print_help
-            shift
```

Comment:
removed this as `print_help` has an `exit 1` at the end and the `shift` is redundant.